### DISCUSSION TITLE
exclude hashes from regex search

### DISCUSSION THREAD

**Discussion by carlo-amz** [sepinf-inc/IPED#1982] [Q&A] [answered]
Hi all!
I have a lot of false positive looking for car plates in the regex search becouse the search is performed in the hashes too.
There is a way to exclude the hashes from the search?
Thank you

**Comment by lfcnassif** [comment]
There is no way today, but seems reasonable to never search for regexes in hash values, we will try to avoid it, thanks for reporting.

**Reply by lfcnassif** [reply, marked as answer]
Hi @carlo-amz, @wladimirleite fixed this here: #1992.
